shell/platform/android/io/flutter/view/FlutterView.java

```diff
@@ -681,10 +681,15 @@ private void handlePlatformMessageResponse(int replyId, byte[] reply) {
         }
     }
 
+    // Called by native to update the semantics/accessibility tree.
     private void updateSemantics(ByteBuffer buffer, String[] strings) {
-        if (mAccessibilityNodeProvider != null) {
-            buffer.order(ByteOrder.LITTLE_ENDIAN);
-            mAccessibilityNodeProvider.updateSemantics(buffer, strings);
+        try {
+            if (mAccessibilityNodeProvider != null) {
+                buffer.order(ByteOrder.LITTLE_ENDIAN);
+                mAccessibilityNodeProvider.updateSemantics(buffer, strings);
+            }
+        } catch (Exception ex) {
+            Log.e(TAG, "Uncaught exception while updating semantics", ex);
```

Comment:
Are there exceptions that are ever actually expected? What are the exceptions we're seeing?